### ISSUE TITLE
Add support for multiple choice questions

### ISSUE THREAD

**Issue by asquare14** [zense/Anonymous-Polling-App#8]
Currently user can vote only for single option.
While creating a poll, a popup button should come up which asks what kind of poll it wants and then direct user to the chosen kind.

**Comment by arkokoley** [comment]
@aSquare14  Any progress on this issue?

**Comment by aSquare14** [comment]
No, No progress. We would like to add this as a learning project in zense and juniors can work on this issue as well as other enhancements. 
Also, for this issue instead of building a separate MVC , we would just use checkboxes instead of radiobuttons !
